File: Hello/app/src/main/java/com/example/hello/MainActivity.java
package com.example.hello;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    EditText usernameText, passwordText;
    Button loginBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        usernameText = findViewById(R.id.usernametxt);
        passwordText = findViewById(R.id.passwordtxt);
        loginBtn = findViewById(R.id.loginbtn);
        loginBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String usernameData = usernameText.getText().toString();
                String passwordData = passwordText.getText().toString();
                Toast.makeText(MainActivity.this, "Username : " + usernameData + " & Password : " + passwordData, Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: Acara9/app/src/main/java/agustacandi/acara9/MainActivity.kt
package agustacandi.acara9

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val toEditText: EditText = findViewById(R.id.to)
        val subjectEditText: EditText = findViewById(R.id.subject)
        val messageEditText: EditText = findViewById(R.id.message)
        val sendBtn: Button = findViewById(R.id.sendBtn)

        sendBtn.setOnClickListener {
            println("Hello")
        }
    }
}